.github/workflows/nightlysnapshot.yml

```diff
@@ -24,8 +26,6 @@ jobs:
       - uses: burrunan/gradle-cache-action@v1
         name: Publish Snapshot
         env:
-          S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ secrets.S3_BUILD_CACHE_ACCESS_KEY_ID }}
-          S3_BUILD_CACHE_SECRET_KEY: ${{ secrets.S3_BUILD_CACHE_SECRET_KEY }}
```

Comment:
S3 access keys are needed only for "push to S3 cache" which happens only from `buildcache.yml`